### ISSUE TITLE
ELI5: project charters

### ISSUE THREAD

**Issue by boneskull** [openjs-foundation/cross-project-council#207]
Charters for individual projects are only [briefly mentioned](https://github.com/openjs-foundation/cross-project-council/blob/d3ae2b9c206e79f6c046f694b63752f24f693743/proposals/stage-1/PROJECT_ONBOARDING/NEW_PROJECT_APPLICATION.md) in the new project application document.  

Can someone please explain:

1. What's a project charter?
1. Why does each project need a charter?
1. What should a project charter would look like for, e.g., Mocha?
1. What about at-large or emeritus projects?

Thanks!

cc @jorydotcom

**Comment by dmethvin** [comment]
I understood the project's charter to be the application as ratified by the CPC. The CPC uses the application to understand how the project is organized and how it governs itself--basically the answers to the Acceptance Criteria for whichever stage the project is in.

Emeritus projects aren't seeking CPC representation and their developers may not be super active, so it's probably not a priority for them to write a charter.

**Comment by mhdawson** [comment]
@boneskull unless @jorydotcom has a simpler one to start from one of the other projects I'd start with https://github.com/nodejs/TSC/pull/698 and remove/simplify content until it makes sense for Mocha.

**Comment by jorydotcom** [comment]
Circling back on this issue since it will be a blocking issue for project onboarding (see #262) - looking back through meeting notes I believe @mkdolan &/or @brianwarner were going to see about getting us a template from other LF projects. If I'm mistaken I am happy to generate a template for us based on Node's charter (JSF charters were formatted more like applications than charters)

**Comment by joesepi** [comment]
@brianwarner will draft charter template

**Comment by jorydotcom** [comment]
Per last week's discussion, @boneskull was satisfied by the answers to the questions and we're addressing the issue template in #279. Closing this issue but feel free to re-open if more documentation is warranted! thanks all <3
